src/capi_frontend/capi.cpp

```diff
@@ -508,6 +508,15 @@ DLL_PUBLIC void OVMS_InferenceRequestDelete(OVMS_InferenceRequest* request) {
     delete reinterpret_cast<InferenceRequest*>(request);
 }
 
+DLL_PUBLIC OVMS_Status* OVMS_InferenceRequestSetCompleteCallback(OVMS_InferenceRequest* req, OVMS_InferenceResponseCompleteCallback_t completeCallback, void* userStruct) {
```

Comment:
```suggestion
DLL_PUBLIC OVMS_Status* OVMS_InferenceRequestSetCompletionCallback(OVMS_InferenceRequest* req, OVMS_InferenceResponseCompleteCallback_t completeCallback, void* userStruct) {
```
Isn't it the name completion callback commonly used in c-apis?